### PR TITLE
create extrapackage.yml

### PR DESCRIPTION
**Pull request by JaseNZC**
Bit of a work in progress I think, It works as is for scene and movies but performer could be added if wanted.

The site layout is bloody terrible and not that easy to work with.

Feel free for anybody to edit away.

### CODE REVIEW

**Comment by bnkai** [comment]
The only thing consistent in the performer page is ethnicity and nationality. Even the d.o.b. is partial (only year). Some aliases and some bio details  are available but only in a few performers. I wouldnt bother with performers unless you want to.

File: scrapers/extrapackage.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+name: extrapackage
+sceneByURL:
+  - action: scrapeXPath
+    scraper: sceneScraper
+    url:
+      - extrapackage.com
```
Comment (bnkai) [inline scrapers/extrapackage.yml:6]: should be 
```yaml
      - extrapackage.com/video
```
to make sure no dvd urls are scraped with the scene scraper

File: scrapers/extrapackage.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+movieByURL:
+  - action: scrapeXPath
+    scraper: movieScraper
+    url:
+      - extrapackage.com
```
Comment (bnkai) [inline scrapers/extrapackage.yml:11]:
```yaml
      - extrapackage.com/dvd
```

File: scrapers/extrapackage.yml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      Details:
+        selector: //div[@class="video-info mt-3"]/p
+      Movies:
+        Name: //div[@class="gblViewInfo"]//div[contains(.,"DVD :")]/a
+      URL:
+        selector: //div[@class="gblViewInfo"]//div[contains(.,"DVD :")]/a/@href
+        postProcess:
+          - replace:
+            - regex: ^
+              with: "http://www.extrapackage.com"
```
Comment (bnkai) [inline scrapers/extrapackage.yml:38]: this needs two spaces in every line to align with the `Name` as it is the URL of  the movie and not the scene

File: scrapers/extrapackage.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  movieScraper:
+    movie:
+      Name:
+        selector: //div[@class="boxGradientTop gradient section"]//p[contains(.,"Information")]
+        postProcess:
+          - replace:
+              - regex: 'Information - \s*'
+                with: ""
+      Date:
+        selector: //td[2]
```
Comment (bnkai) [inline scrapers/extrapackage.yml:57]:
```yaml
        selector: //td[@align="right"]
```
simpler

File: scrapers/extrapackage.yml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        postProcess:
+          - parseDate: 2006-01-02
+      Duration:
+        selector: //td/div[1]
```
Comment (bnkai) [inline scrapers/extrapackage.yml:61]:
```yaml
      Duration:
        selector: //td/div
        postProcess:
          - replace:
              - regex: $
                with: ":00"
```
The movie duration i think expects seconds so adding the `:00` part should make the UI autocorrect it
I dont think the `[1]` was  needed for the div? If it was readd it

File: scrapers/extrapackage.yml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+      Studio:
+        Name:
+          selector: //li[contains(.,"Studio")]/a
+          postProcess:
+            - replace:
+                - regex: 'Studios\s*'
+                  with: ""
```
Comment (bnkai) [inline scrapers/extrapackage.yml:68]: the same selector as in the scenes is working here 
```yaml
      Studio:
        Name:
          selector: //div[@class="gblViewInfo"]//li[contains(.,"Studio")]/a
```
The one you have matches the Studio from the navbar

**Comment by bnkai** [comment]
can you also rename the file to `ExtraPackage.yml` ?

**Comment by JaseNZC** [comment]
All completed. :-)

**Pull request merged**